rust/relay/src/lib.rs

```diff
@@ -0,0 +1 @@
+pub mod server;
```

Comment:
Do we have a separate lib from the binary to have tests and docs?

Reply:
Yes. We can't have crate-level tests unless the crate is a library.